sound/hybrid_font.h

```diff
@@ -120,7 +132,49 @@ class HybridFont : public SaberBase {
       PlayPolyphonic(effect);
     }
   }
+  
+  void StartSwing() override {
+    if (!guess_monophonic_) {
+      if (swing_player_) {
+        // avoid overlapping swings, based on value set in ProffieOSSwingOverlap.  Value is
+        // between 0 (no overlap) and 1.0 (full overlap)
+        if (swing_player_->pos() / swing_player_->length() >= config_.ProffieOSSwingOverlap) {
+          RefPtr<BufferedWavPlayer> overlap_swing = swing_player_;
+          swing_player_ = PlayPolyphonic(&swng);
+          overlap_swing->set_fade_time(overlap_swing->length() - overlap_swing->pos());
+          overlap_swing->FadeAndStop();
+          overlap_swing.Free();
+        }
+      }
+      else if (!swing_player_) {
+        swing_player_ = PlayPolyphonic(&swng);
+        swing_player_->set_volume_now(0);
+      }
+    } else {
+      PlayMonophonic(&swing, &hum);
+    }
+  }
 
+  float SetSwingVolume(float swing_strength, float mixhum) override {
+    if(swing_player_) {
+      if (swing_player_->isPlaying()) {
+        float accent_volume = powf(swing_strength, config_.ProffieOSSwingVolumeSharpness) * config_.ProffieOSMaxSwingVolume;
+        swing_player_->set_volume(accent_volume);
+        mixhum = mixhum - mixhum * (config_.ProffieOSSmoothSwingDucking * swing_strength);
```

Comment:
seems like we should use accent_volume, not swing strength here
I'm still not convinced we need this.

Reply:
I'll test with accent_volume and see how it works.  I think we want this parameter for a couple reasons:

Testers have reported audio issues in a couple of circumstances:
1) the volume crackling that appears to be fixed or at least reduced frequency by setting player volume to 0
2) Additional volume crackling issues on loud or bass heavy accent swings that are 'fixed' by reducing accent swing volume.  This parameter lets users and font makers choose which trade-off to make: they can reduce accent swing volume using the max parameter or they can reduce swing pair volume during swings using the ducking parameter.

Reply:
using accent_Volume works well, and via testing the ducking parameter is required to reduce peaks below 32k